server/views/components/drawer/drawer.njk

```diff
@@ -0,0 +1,13 @@
+<div class="drawer js-drawer">
+  <button class="drawer__header flex flex--v-center flex--h-space-between plain-button {{ {s:0} | spacingClasses({padding: ['left', 'right']}) }} {{ {s:2} | spacingClasses({padding: ['top', 'bottom']}) }} js-show-hide-trigger {{ {s:'LR2'} | fontClasses }}">
+    <span class="drawer__heading">{{ model.heading }}</span>
+    <span class="drawer__icon">
+      {% icon 'actions/more-2', null, ['icon--match-text'] %}
+    </span>
+  </button>
+  <div class="drawer__body {{ {s:2} | spacingClasses({padding: ['top', 'bottom']}) }} js-show-hide-drawer">
+    {{ model.body }}
+  </div>
+  <hr class="divider divider--black divider--keyline" />
+</div>
```

Comment:
Sorry, didn't mean for you to review while it was WIP (without tracking) – have added now.